charts/pgadmin4/README.md

```diff
@@ -96,4 +96,34 @@ $ helm install runix/pgadmin4 --name my-release -f values.yaml
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+## Examples
+
+### Ingress with NGINX controller
+
+Deploy pgAdmin4 under particular domain name and URL path.
+
+Create/edit the `values.yaml` file with following ingress configuration:
```

Comment:
Personally I prefer configuration examples within the values.yaml with reference links, e.g.:
```
## Node labels for pgAdmin4 pod assignment
## Ref: https://kubernetes.io/docs/user-guide/node-selection/
##
```

So in this case, https://kubernetes.io/docs/concepts/services-networking/ingress/ should just be added towards the ingress part perhaps?

In your example you make the assumption a person uses nginx as ingress.class. So I kinda like to leave this as it is within the current values.yaml:

```
ingress:
  ## If true, pgAdmin4 Ingress will be created
  ##
  enabled: false

  ## pgAdmin4 Ingress annotations
  ##
  annotations: {}
    # kubernetes.io/ingress.class: nginx
    # kubernetes.io/tls-acme: "true"

  ## pgAdmin4 Ingress hostnames with optional path
  ## Must be provided if Ingress is enabled
  hosts:
    - host: chart-example.local
      paths: []

  ## pgAdmin4 Ingress TLS configuration
  ## Secrets must be manually created in the namespace
  tls: []
  #  - secretName: chart-example-tls
  #    hosts:
  #      - chart-example.local
```

Perhaps just add a link there to what you attempt to achieve?

Reply:
Thanks for your comment, @rowanruseler .

I've put that here because I just found it difficult to configure pgAdmin to be accessible to on some particular URL via Ingress. Need to look into the app manual, nginx docs, etc.
Setting ingress like this is pretty common.

Mentioned that the example is for NGINX ingress controller only, and the aim is to have simple copy-paste-and-edit snippet as a quick solution.
That is more specific to pgAdmin, since it needs custom header and even redirects despite only setting the ingress.

I thought about putting it to `values.yaml`, but that would look ugly in my opinion and hard to get how to put it all together. The README is usually being looked first, so having some common recipe there is very convenient.

But if you'd like to move it or duplicate it in the `values.yaml` with some more detailed comments, I could certainly do that.

Reply:
Apologies for my late reply, apparently I left it on pending.

Reply:
Thanks for your feedback, @rowanruseler . No worries.
Then I leave the example in README and update the `values.yaml`.

Reply:
Done, removed.